prime-router/build.gradle.kts

```diff
@@ -454,7 +455,19 @@ tasks.register("quickPackage") {
     tasks["dokkaHtml"].enabled = false
 }
 
+/**
+ * Docker services needed for running Dockerless
+ */
+dockerCompose {
+    projectName = "prime-router"
+    useComposeFiles.addAll("docker-compose.yml")
+    startedServices.addAll("sftp", "ftps", "soap-webservice", "vault", "azurite")
```

Comment:
The other startup methods wait for vault to populate the env var file. Does that happen here, or is that wait unnecessary?

Reply:
That is not needed here.  This is specifically to start the necessary services, not to set them up.  This replicates the manual process only.

Reply:
The only thing this replaces is having to run that docker compose up with the specific container names